src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx

```diff
@@ -10,33 +10,32 @@ const TermsOfUse = () => {
       <div className='second-box-terms'>
         <div className='box-terms'>
```

Comment:
Why is file name `FrequentlyAskedQuestions.jsx`, and component inside `TermsOfUse`

Reply:
The text has changed and is now in every component it belongs to

Reply:
I see here that nothing change, files `FrequentlyAskedQuestions` and `PrivacyNotes` still have same component TermsOfUse. Have you changed that in another branch or?

Reply:
Changes are found on the fixed impressions component